### PR TITLE
Add debit, billable_xxx, roaming, and subtype to the usage record attributes

### PR DESCRIPTION
**Pull request by arvanasse**
The reactor end point already supports all the 'new' attributes except 'subtype'.  Adding them to the parser's UsageRecord class will allow atom-api to access them.

### CODE REVIEW

File: lib/conduit/reactor/parsers/usage_record.rb [hunk 1]
```diff
@@ -1,18 +0,0 @@
-require_relative 'base'
-
-module Conduit::Driver::Reactor
-  class UsageRecord < Struct.new(:id, :service_type, :sourcefile, :bill_number
-    :create_time, :usage_time, :from_number, :to_number, :data_octets, :location,
-    :call_seconds)
-
-    def used_at
```
Comment (pedromartinez) [inline lib/conduit/reactor/parsers/usage_record.rb]: Is this not needed?

Comment (arvanasse) [inline lib/conduit/reactor/parsers/usage_record.rb]: No, it's defined in the parser as well (https://github.com/conduit/conduit-reactor/blob/ATOM-2786_add_debit_billable_and_roaming_attributes_to_usage/lib/conduit/reactor/parsers/query_usage.rb#L20-30), and the parser version is better sync'ed to the parser.

Comment (pedromartinez) [inline lib/conduit/reactor/parsers/usage_record.rb]: k

**Comment by arvanasse** [comment]
Merged #49

**Pull request merged**